automation/taskcluster/lib/tasks.py

```diff
@@ -185,6 +185,35 @@ def _craft_clean_gradle_task(
             treeherder=treeherder,
         )
 
+    def craft_dependencies_task(self):
+        # Output the dependencies to an artifact.  This is used by the
+        # telemetry probe scraper to determine all of the metrics that
+        # &browser might send (both from itself and any of its dependent
```

Comment:
Unnecessary `&` here :)

Reply:
I think that's for reference(!) browser :D